src/uu/ls/src/ls.rs

```diff
@@ -1958,7 +1958,14 @@ fn sort_entries(entries: &mut [PathData], config: &Config, out: &mut BufWriter<S
             )
         }),
         Sort::Size => entries.sort_by_key(|k| Reverse(k.md(out).map(|md| md.len()).unwrap_or(0))),
-        // The default sort in GNU ls is case insensitive
+        // The default sort in GNU `ls` is Case-Sensitive
+        // 
+        // Platform with default Case-Sensitive `ls` Sorting (Most GNU)
+        // - Most Linux Distributions
+        // - Unix-Based Systems (e.g., macOS)
+        //
+        // Platform with default Case-Insensitive `ls` Sorting
+        // - Microsoft Windows
```

Comment:
I think this is a too verbose comments. 
If you are reading the source of this implementation of ls, you probably know that Windows is case insensitive and Unix-based system's are. No ? :)

Reply:
Probably not :')

I just stumbled onto this code where I need to implement several sort functions (extensions) into my project. I agree that this comment is too verbose. But perhaps the comment needs to state that  `Case-Sensitive` is GNU default instead of `Case-insensitive`.